runtime-modules/working-group/src/lib.rs

```diff
@@ -1376,7 +1432,7 @@ impl<T: Trait<I>, I: Instance> Module<T, I> {
         WorkerById::<T, I>::iter()
             .filter_map(|(worker_id, worker)| {
                 if let Some(started_leaving_at) = worker.started_leaving_at {
-                    if started_leaving_at + worker.job_unstaking_period >= Self::current_block() {
+                    if started_leaving_at + worker.job_unstaking_period <= Self::current_block() {
```

Comment:
I don't understand this change, isn't the unstaking period finished after `job_unstaking_period`?

Reply:
It's easier to reason about if we rewrite the condition: `if current_block >= last_worker_date`

Reply:
But this condition was correct before? It seems that before it was `if current_block <= worker_leaving_block` or there was some other change related to this?

Reply:
Added an additional test to verify correct work of the `job_unstaking_period` parameter.